bootcamp/news/urls.py

```diff
@@ -16,4 +16,5 @@
     url(
         r"^update-interactions/$", views.update_interactions, name="update_interactions"
     ),
+    url(r'^(?P<pk>[\w.@+-]+)/$', views.news, name='news'),
```

Comment:
What is the purpose of this URL pattern? It looks like it is to show individual news on a different page, which in time render useless the original feature. If that's the intent, what is the logic behind it? And if it only makes a call to a single element, is it necessary to make a pattern that complex? It looks like the simpler pattern added with `path()` will suffice.

Reply:
since in this version we are dealing with UUID instead of simple id, this pattern allows to show that uuid. But maybe could be simplified

Reply:
I haven't had the time of migrating all the URL from the old pattern using `url()` to the new method given in Django2 using `path()`, which actually has a `<uuid>` identifying structure.